### PR TITLE
Group1

### PR DESCRIPTION
**Pull request by Jvarkul**

### CODE REVIEW

**Review by aidkvndfj — CHANGES_REQUESTED** [review]
image
There seems to be some conflicts with what you've requested, and the changes on master. Please resolve these issues and we will do from there. I recommended pulling the changes from master to the branch, and we can go from there. Please let us know if you need assistance.

**Review by cc0407 — CHANGES_REQUESTED** [review]
Can you please make a test handler in your ReadInputTests.c and .h file that run all 6 of your tests. Once you've implemented that add it to main.c inside of runAllTests().

Also, remove "group1.h" from groupIncludes.h and add your .h files to groupIncludes.h and groupTestIncludes.h

After that just give it a quick once over with `make`, `./main`, `./main -t`, and the valgrind command from the readMe to make sure it runs without any warnings or errors!

File: ReadInput/ReadInputTests.h [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include "../main.h"
+#include "ReadInput.h"
+
+char test1(char *, char *, char *);
```
Comment (andrewlinington) [inline ReadInput/ReadInputTests.h:4]: please rename these to be more group-specific like ReadInputTest1

File: inputTransformation.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+setGlobalScale=1
+setXScale=1
+setYScale=1
+setZScale=1
+setXRotation=0
+setYRotation=0
+setZRotation=0
+setXTranslation=0
+setYTranslation=0
+setZTranslation=0
```
Comment (andrewlinington) [inline inputTransformation.txt:10]: seems the shear groups were forgotten when i sent the sample so they could be added in here

File: main.c [hunk 3]
```diff
@@ -113,15 +81,14 @@ int main(int argc, char** argv) {
...
         free(inputShape);
+        
+        free(inputFileName);
+        free(outputFileName);
     }
     else {
-        // fprintf(stderr, "Format %s <input file> <output file>", argv[0]);
-        fprintf(stderr, "Format: %s <output file>\n", argv[0]);
-        free(fileName);
+        fprintf(stderr, "Format %s <input file> <transformation file> <output file>", argv[0]);
```
Comment (andrewlinington) [inline main.c:89]: add newline

File: zPlaneReflection/zPlaneReflectionTests.h [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#ifndef Z_PLANE_REFLECTION_TESTS_H
```
Comment (andrewlinington) [inline zPlaneReflection/zPlaneReflectionTests.h:1]: delete this file

File: main.c [hunk 5]
```diff
@@ -4,53 +4,21 @@
...
-        setXRotation(0);
-        setYRotation(0);
-        setZRotation(0);
-        setXTranslation(0);
-        setYTranslation(0);
-        setZTranslation(0);
-        setXShear(0);
-        setYShear(0);
-        setZShear(0);
-
```
Comment (andrewlinington) [inline main.c:52]: I know it isn't here but you need to include your tests at the bottom if you want to submit them (if not still add them but comment them out so we know you have them

**Review by andrewlinington — CHANGES_REQUESTED** [review]
a few issues need to be resolved but they are mostly small issues when we get to actually merging to master

**Review by andrewlinington — APPROVED** [review]
Looks great good to merge. We Will need to fix tests and adjust the rotation values by multiplying by (pi/180)

**Review by vdixon531 — APPROVED** [review]

**Pull request merged**